feg/gateway/services/n7_proxy/servicers/session_controller.go

```diff
@@ -0,0 +1,211 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+
+	fegprotos "magma/feg/cloud/go/protos"
+	"magma/feg/gateway/policydb"
+	n7_client "magma/feg/gateway/sbi/specs/TS29512NpcfSMPolicyControl"
+	"magma/feg/gateway/services/n7_proxy/metrics"
+	"magma/lte/cloud/go/protos"
+	orcprotos "magma/orc8r/lib/go/protos"
+)
+
+type CentralSessionController struct {
+	policyClient  *n7_client.ClientWithResponses
+	dbClient      policydb.PolicyDBClient
+	cfg           *SessionControllerConfig
+	healthTracker *metrics.SessionHealthTracker
+}
+
+type SessionControllerConfig struct {
+	PCFConfig      *PCFConfig
+	RequestTimeout time.Duration
+	DisableN7      bool
+}
+
+func newSessionControllerConfig(config *N7ProxyConfig) *SessionControllerConfig {
+	return &SessionControllerConfig{
+		PCFConfig:      config.Server,
+		RequestTimeout: 3 * time.Second,
```

Comment:
Try not to hardcode values in the middle of the code 

Use constants 

https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=feg%2Fgateway%2Fgtp%2Fgtp_client.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=31&start_col=12&end_row=31&end_col=12